lib/nermesterts/game_picker.ex

```diff
@@ -26,6 +26,8 @@ defmodule GamePicker do
       nil
   """
   def pick_game(games_list, num_players) do
+    :rand.seed(:exsplus, DateTime.utc_now |> DateTime.to_date |> Date.to_erl)
```

Comment:
@daveshah I'd like to respectively disagree that this is non-deterministic. Given the same seed and the same pseudorandom algorithm, you should always get the same output. The seed is what gives the RNG its non-determinism (which we're taking away)